velox/common/hyperloglog/Murmur3Hash128.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cstdint>
+
+namespace facebook::velox::common::hll {
+
+int64_t murmur3Hash64(const void* data, int32_t length, int64_t seed);
```

Comment:
The correct name should be `Murmur3Hash128::hash64`, it's the 64 most significant bits of the Murmur3 128-bit hash.  Also add a link to the Java version: https://github.com/airlift/slice/blob/master/src/main/java/io/airlift/slice/Murmur3Hash128.java